server/datastore/mysql/mdm.go

```diff
@@ -61,7 +61,7 @@ SELECT
     wmc.command_uuid,
     COALESCE(NULLIF(wmcr.status_code, ''), 'Pending') as status,
     COALESCE(wmc.updated_at, wmc.created_at) as updated_at,
-    wmc.target_loc_uri as request_type,
+    COALESCE(NULLIF(wmc.target_loc_uri, ''), 'InstallProfile') as request_type,
```

Comment:
@mna to be honest I'm not sure what's the best thing to do here. Right now the only windows command that can have an empty LocURI is the `Atomic` we use for profiles, but... what if there's a bug? or if this changes?

on the other hand, I don't want to over-complicate things.

do you have any thoughts?

Reply:
@roperzh 

> 1. It's confusing, InstallProfile is very macOS specific, there's no similar concept in Windows
> 2. Feels like it's "view" logic that we shouldn't bake into the DB. What if product decides to change the wording? another DB migration?

I very much agree with that, 1. was my initial reaction and 2. is very true. To avoid being too brittle/error-prone, I feel like this should rely on something that we control and that we can tell 100% that this was coming from a profile, instead of relying on the absence of a target URI.

That being said - as you say, to avoid over-complicating things -, we may not have that kind of thing handy at the moment, so if we can get away with just doing this in `fleetctl` when formatting the result, I think that would be my number 1 choice (well, technically 2 I guess)? And if it's not good enough in the future, we haven't lost a ton of time doing something complex and it's an easy change. If we absolutely want that "InstallProfile" to be part of the API response payload, then doing it in the query like that would be similarly simple to evolve in the future if needed.